### ISSUE TITLE
Time to get a new host?

### ISSUE THREAD

**Issue by davl3232** [willy-vvu/reveal.js#15]
<img width="483" alt="screenshot 2015-12-24 05 50 45" src="https://cloud.githubusercontent.com/assets/4100566/11993732/834a3b5e-aa02-11e5-86b4-c6e6f0840ac6.png">

**Comment by willy-vvu** [comment]
Yeah... Haven't touched this project in ages. Has it really been three years?

**Comment by willy-vvu** [comment]
I should be more active on this, even if it involves switching the branch to gh-pages and changing all references. But at this point, I'd also consider this project in an archival state, with at least 1,300 upstream commits of reveal.js...

**Comment by davl3232** [comment]
Thanks for the contribution anyway.
